test/spec/services/apiServiceSpec.js

```diff
@@ -234,6 +235,50 @@ describe("APIService", function(){
       });
     }));
   });
-  */
+
+
+  describe('#availableKinds', function() {
+    // at minimum, prove with tests that it is deduplicating by ignoring openshift!
+    it('should return list of kinds that are scoped to a namespace by default', function() {
+      var namespacedKinds = APIService.availableKinds();
+      var expectedNames = [
+        'Binding', 'ConfigMap', 'Endpoints', 'Event', 'LimitRange', 'PersistentVolumeClaim', 'Pod', 'PodTemplate',
+        'ReplicationController', 'ResourceQuota', 'Secret', 'ServiceAccount', 'Service', 'StatefulSet', 'DeploymentConfig',
+        'LocalSubjectAccessReview', 'LocalResourceAccessReview', 'LocalSubjectAccessReview', 'Policy', 'PolicyBinding',
+        'ResourceAccessReview', 'RoleBindingRestriction', 'RoleBinding', 'Role', 'SelfSubjectRulesReview',
+        'SubjectAccessReview', 'SubjectRulesReview', 'BuildConfig', 'Build', 'DaemonSet', 'Deployment',
+        'HorizontalPodAutoscaler', 'Ingress', 'Job', 'NetworkPolicy', 'ReplicaSet', 'ReplicationControllerDummy',
+        'ImageStreamImage', 'ImageStreamImport', 'ImageStreamMapping', 'ImageStream', 'ImageStreamTag', 'EgressNetworkPolicy',
+        'PodDisruptionBudget', 'AppliedClusterResourceQuota', 'Route', 'PodSecurityPolicyReview',
+        'PodSecurityPolicySelfSubjectReview', 'PodSecurityPolicySubjectReview', 'Template', 'TemplateInstance'
+      ];
```

Comment:
This test is going to break often because the kinds will change. I'd rather not maintain this list if we can avoid it.

Is there another way to test? Maybe check that a subset exists and that OpenShift dupes are not there?